docs/faq/install.txt

```diff
@@ -49,7 +49,8 @@ Django version Python versions
 1.8            2.7, 3.2 (until the end of 2016), 3.3, 3.4, 3.5
 1.9, 1.10      2.7, 3.4, 3.5
 1.11           2.7, 3.4, 3.5, 3.6
-2.0            3.5+
+2.0            3.4, 3.5, 3.6
+2.1            3.5, 3.6
```

Comment:
python 3.7 is supposed to be released by june 2018, which might make it possible to support 3.7 in django 2.1?